src/Protocol/ActivityPub/Receiver.php

```diff
@@ -1419,18 +1414,7 @@ private static function processObject($object)
 
 		if (in_array($object_data['object_type'], ['as:Audio', 'as:Video'])) {
 			$object_data['alternate-url'] = self::extractAlternateUrl($object['as:url'] ?? []) ?: $object_data['alternate-url'];
-
-			$attachments = self::processAttachmentUrls($object['as:url'] ?? []);
-			foreach ($attachments as $type => $attachment) {
-				$object_data['attachments'][] = [
-					'type' => $type,
-					'mediaType' => $attachment['type'],
-					'height' => $attachment['height'],
-					'size' => $attachment['size'],
-					'name' => '',
-					'url' => $attachment['url']
-				];
-			}
+			$object_data['attachments'] = self::processAttachmentUrls($object['as:url'] ?? []);
```

Comment:
We also assign the "attachments" field here: https://github.com/friendica/friendica/blob/develop/src/Protocol/ActivityPub/Receiver.php#L1403

We mustn't replace but add the values here.